filebeat/reader/readfile/limit.go

```diff
@@ -37,6 +38,13 @@ func NewLimitReader(r reader.Reader, maxBytes int) *LimitReader {
 func (r *LimitReader) Next() (reader.Message, error) {
 	message, err := r.reader.Next()
 	if len(message.Content) > r.maxBytes {
+		diff := len(message.Content) - r.maxBytes
+		message.AddFields(common.MapStr{
+			"log": common.MapStr{
+				"truncated": diff,
+			},
+		})
```

Comment:
What if the contents is already truncated (multiline is applied before limit reader)? We should add the counters, no?